rpc/README.md

```diff
@@ -1372,6 +1373,51 @@ Response
 }
 ```
 
+#### Method `get_median_fee_rate`
+* `get_median_fee_rate(blocks_to_scan, num_transactions)`
+    * `blocks_to_scan`: [`Uint32`](#type-uint32) `|` `null`
+    * `num_transactions`: [`Uint32`](#type-uint32) `|` `null`
```

Comment:
wrong params name?
```suggestion
    * min_transactions`: [`Uint32`](#type-uint32) `|` `null`
```

Reply:
Missing a \` in the suggested change.

Reply:
i change the param name to:  `blocks_to_scan` and `min_transactions`